src/lib/components/correction/correction_grading_table.dart

```diff
@@ -0,0 +1,250 @@
+import 'package:collection/collection.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:modal_bottom_sheet/modal_bottom_sheet.dart';
+import 'package:schoolexam_correction_ui/blocs/remark/remark.dart';
+
+/// This widget allows for editing of the grading table that should be assigned to the exam
```

Comment:
Das schon ein Bomben-Widget, uff. Kann man das evtl. in etwas kleinere Pakete unterteilen?

Reply:
Habe den Header und die Action Bar extrahiert.